EssentialApp/EssentialAppTests/FeedAcceptanceTests.swift

```diff
@@ -83,6 +99,9 @@ class FeedAcceptanceTests: XCTestCase {
 		case "/essential-feed/v1/feed":
 			return makeFeedData()
 
+		case "/essential-feed/v1/image/A28F5FE3-27A7-44E9-8DF5-53742D0E4A5A/comments":
```

Comment:
My last comment was just an example - you need to update the image `id` matching the selection in the test.

In the test method, you're selecting the first image. So you need to use the first image `id` to stub its comments.

The first image `id` defined in the data below is `2AB2AE66-A4B7-4A16-B374-51BBAC8DB086`.

So you need to update the case here to use the right ID:

```suggestion
		case "/essential-feed/v1/image/2AB2AE66-A4B7-4A16-B374-51BBAC8DB086/comments":
```